### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by paulb777**
## Description of the change
README.md updates

## Motivation
Improve docs

## Type of change
Choose one: Documentation

See preview at https://github.com/google/generative-ai-swift/blob/8e5fce6aaecc5c22b2d5e9c702b20313d90d9726/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,86 @@
 # Google AI Swift SDK
 
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fgoogle%2Fgenerative-ai-swift%2Fbadge%3Ftype%3Dswift-versions)](https://swiftpackageindex.com/google/generative-ai-swift)
+[![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2Fgoogle%2Fgenerative-ai-swift%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/google/generative-ai-swift)
```
Comment (ryanwilson) [inline README.md:4]: I don't think these are accurate, I'd rather wait until they're updated appropriately. Right now it shows Swift 5.7+ which I believe we're only 5.9+, and it says visionOS, Linux, watchOS are supported which they are not (untested at least)

File: README.md [hunk 2]
```diff
@@ -1,17 +1,86 @@
...
+- Generate text from text-and-images input (multimodal)
+- Build multi-turn conversations (chat)
+
+For example, with just a few lines of code, you can access Gemini's multimodal capabilities to
+generate text from text-and-image input:
+
+```swift
+let model = GenerativeModel(name: "gemini-pro-vision", apiKey: "YOUR_API_KEY")
+let cookieImage = UIImage(...)
+let prompt = "Give me a recipe for this:"
```
Comment (ryanwilson) [inline README.md:24]:
```suggestion
let prompt = "Do these look store-bought or homemade?"
```

File: README.md (hunk 1 above)
Comment (paulb777) [reply README.md]: OK. Removed for now

File: README.md (hunk 2 above)
Comment (paulb777) [reply README.md]: Done.

**Review by ryanwilson — APPROVED** [review]

**Pull request merged**